bcda/utils/common.go

```diff
@@ -72,3 +72,19 @@ func IsUUID(s string) bool {
 	re := regexp.MustCompile("^[a-fA-F0-9]{8}(?:-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$")
 	return re.MatchString(s)
 }
+
+// CloseSafely will close a file and log any errors
+func CloseSafely(f *os.File) {
+	if ferr := f.Close(); ferr != nil {
+		logrus.Error(ferr)
+	}
+}
+
+// CloseSafelyAndReturnError will close a file, log, and return any errors
+func CloseSafelyAndReturnError(f *os.File) (err error) {
```

Comment:
Is this function necessary?  Previously, we were completely losing the error when we called `defer f.Close()`.  No need to return an error now, unless I'm missing a scenario

Reply:
No, it isn't necessary. If we are losing the error then no need to accommodate this function. I was trying to do a lot more than was called for!